reference/forms/types/dateinterval.rst

```diff
@@ -0,0 +1,325 @@
+
+DateIntervalType Field Type
+===========================
+
+.. versionadded:: 3.2
+   The DateIntervalType field type was introduced in Symfony 3.2.
+
+This field type allows the user to modify data that represents a specific
+date interval. It can be rendered as a integer or text input or select tags.
+
+The underlying format of the data can be a ``DateInterval`` object,
+a `ISO 8601`_ duration string (e.g. ``P1DT12H``) or an array.
+
++----------------------+-----------------------------------------------------------------------------------+
+| Underlying Data Type | can be ``DateInterval``, string or array (see the ``input`` option)               |
++----------------------+-----------------------------------------------------------------------------------+
+| Rendered as          | single text box or up to six select, text or integer boxes plus optional checkbox |
++----------------------+-----------------------------------------------------------------------------------+
+| Options              | - `days`_                                                                         |
+|                      | - `placeholder`_                                                                  |
+|                      | - `hours`_                                                                        |
+|                      | - `input`_                                                                        |
+|                      | - `minutes`_                                                                      |
+|                      | - `months`_                                                                       |
+|                      | - `seconds`_                                                                      |
+|                      | - `weeks`_                                                                        |
+|                      | - `widget`_                                                                       |
+|                      | - `with_days`_                                                                    |
+|                      | - `with_hours`_                                                                   |
+|                      | - `with_invert`_                                                                  |
+|                      | - `with_minutes`_                                                                 |
+|                      | - `with_months`_                                                                  |
+|                      | - `with_seconds`_                                                                 |
+|                      | - `with_weeks`_                                                                   |
+|                      | - `with_years`_                                                                   |
+|                      | - `years`_                                                                        |
```

Comment:
Back then I used the datetime doc as base for my documentation which had it that way. Some types are alphabetically, others aren't.
Maybe that's something to rework for all types in another PR?